### PR TITLE
chore(precompile): fix incorrect comment

### PR DESCRIPTION
**Pull request by taeguk**

### CODE REVIEW

File: cosmos/precompile/bank/bank.go [hunk 1]
```diff
@@ -202,7 +202,7 @@ func (c *Contract) GetSendEnabled(
 	return res.SendEnabled[0].Enabled, nil
 }
 
-// Send implements `send(address,(uint256,string))` method.
+// Send implements `send(address,(uint256,string)[])` method.
```
Comment (itsdevbear) [inline cosmos/precompile/bank/bank.go:205]: string[])

Comment (taeguk) [reply cosmos/precompile/bank/bank.go:205]: what do you mean?
I think what I fixed is correct. It should express an array of `Coin`, which includes an amount and a denom.

Comment (itsdevbear) [reply cosmos/precompile/bank/bank.go:205]: oh mb i'm goofy

**Review by itsdevbear — APPROVED** [review]

**Comment by itsdevbear** [comment]
@calbera @transmissions12  can we get approvals please

**Comment by calbera** [comment]
linter not passing

**Review by itsdevbear — APPROVED** [review]

**Pull request merged**